modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-evp/osb-site-initializer-evp-test/src/testFunctional/tests/EVPOrganization.testcase

```diff
@@ -61,6 +57,34 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-172949. Verify that is possible finance user can edit the Organization with 'Awaiting Finance Review' status"
```

Comment:
Hey @joelrocha68, what you think about this new Scenario description
`"LPS-172949. Verify that is possible finance user can edit the Organization with 'Awaiting Finance Review' status"`